tests/instancetype/instancetype.go

```diff
@@ -1020,10 +1019,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		)
 	})
 
-	Context("instance type with dedicatedCPUPlacement enabled", func() {
-		BeforeEach(func() {
-			checks.SkipTestIfNoCPUManager()
-		})
+	Context("instance type with dedicatedCPUPlacement enabled", decorators.RequiresTwoWorkerNodesWithCPUManager, func() {
```

Comment:
I believe this makes the test more expensive.
It used to require 1 node with CPUManager (i.e. a 2-node cluster), this now makes it require 2 (i.e. a 3-node cluster).

Reply:
On CI we spin up 3 nodes cluster aren't we?

Reply:
I believe only sig-compute-migrations has 3 nodes, the others lanes have 2.

Reply:
Introduced `RequiresNodeWithCPUManager`